src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -4,7 +4,7 @@
     from-bottom
     has-close-button
     class="multisig-vault-create"
-    @close="reject()"
+    @close="reject"
```

Comment:
I know that it's possible to not use the parentheses but actually it's a bad pattern, as this kind of changes ends up in passing events into functions as arguments.
So for example when you do this:
```
@click="someAction"
```
what actually happens is
```
@click="(event) => someAction(event)"
```
and for the `@click` the event is `MouseEvent`. This is why I always suggest to leave empty paren to block the possibility of passing unintended arguments.